### ISSUE TITLE
Add separte Login for PARSE.com connection

### ISSUE THREAD

**Issue by EbereAko** [LuiAlva/SwirlyTap#54]
one for Level Play and one for Time mode

**Comment by Diana-Pham** [comment]
Can't these be linked?

**Comment by LuiAlva** [comment]
Yeah, you should put the login with the others. There's also an issue with using both a local and internet database at the same time. If you erase the data off the phone, the local profile will be deleted, so when the you try to log in, it will fail for the local because the profile was deleted, while the internet one will sign in because the profile will be stored on the online database.
